sdk/communication/Azure.Communication.Common/tests/CommunicationRecordedTestSanitizer.cs

```diff
@@ -13,6 +14,7 @@ public class CommunicationRecordedTestSanitizer : RecordedTestSanitizer
         private static readonly Regex _azureResourceRegEx = new Regex(@"[^/]+?(?=(.communication.azure))", RegexOptions.Compiled);
         private static readonly Regex _identityInRouteRegEx = new Regex(@"(?<=identities/)([^/]+)", RegexOptions.Compiled);
         private static readonly Regex _phoneNumberRegEx = new Regex(@"[\\+]?[0-9]{11,15}", RegexOptions.Compiled);
```

Comment:
your baseline seems off of master. The phone number sanitizations are moved to the PhoneNumbers.Tests project

Reply:
I have it rebased.